### PR TITLE
Sprint #61 update.

### PR DESCRIPTION
**Pull request by cornedenhollander**
scrnli_09_06_2021_10-50-26

### CODE REVIEW

File: _site/roadmap.html [hunk 1]
```diff
@@ -45,36 +45,15 @@ <h2 class="text-3xl font-display text-sandwave-500 mb-4">Now</h2>
...
-                                            DNS Management
-                                        </span>
-                                    </div>
-                                    <div class="block mt-2">
-                                        <p class="h3 font-semibold text-black">
-                                            Setup redirects
-                                        </p>
-                                        <p class="mt-3 text-sm text-gray-darker">
-                                            Enable customers to setup redirects from one domain to another.
+                                            Create a single hub where customers can log in after which they will be routed to the applicable Control Panel.
```
Comment (AvOeffelen) [inline _site/roadmap.html:56]: "Create a landing page where customers can log in after which they will be routed to the applicable Control panel"

File: _site/roadmap.html [hunk 2]
```diff
@@ -145,117 +113,115 @@ <h2 class="text-3xl font-display text-sandwave-500 mb-4">Later</h2>
...
+                <div class="py-3">
+                    <div class="rounded-lg shadow-lg overflow-hidden py-1">
+                        <div class="flex-1 bg-white px-6 py-2 justify-between">
+                            <div class="flex-1">
+                                <div class="block mt-2">
+                                    <p class="h3 font-semibold text-black">
+                                        Bank tool
+                                    </p>
+                                    <p class="mt-3 text-sm text-gray-darker">
+                                        Setting up a bank tool that will take care of the manual transfers done to pay invoices.
```
Comment (AvOeffelen) [inline _site/roadmap.html:148]: manual will be SEPA right

File: _site/roadmap.html [hunk 3]
```diff
@@ -145,117 +113,115 @@ <h2 class="text-3xl font-display text-sandwave-500 mb-4">Later</h2>
...
+                <div class="py-3">
+                    <div class="rounded-lg shadow-lg overflow-hidden py-1">
+                        <div class="flex-1 bg-white px-6 py-2 justify-between">
+                            <div class="flex-1">
+                                <div class="block mt-2">
+                                    <p class="h3 font-semibold text-black">
+                                        Recognize premium domains
+                                    </p>
+                                    <p class="mt-3 text-sm text-gray-darker">
+                                        Making sure premium domains aren't sold for regular prices.
```
Comment (AvOeffelen) [inline _site/roadmap.html:180]: are not being ??

File: _site/roadmap.html [hunk 4]
```diff
@@ -266,26 +232,61 @@ <h2 class="text-3xl font-display text-sandwave-500 mb-4">Recently released</h2>
                             <div class="flex-1">
                                 <div class="block mt-2">
                                     <p class="h3 font-semibold text-black">
-                                        Sitebuilder integration
+                                        Install Wordpress
                                     </p>
                                     <p class="mt-3 text-sm text-gray-darker">
-                                        Enable customers to use a sitebuilder for their domain.
+                                        Enable customers to easily install Wordpress on their hosting package. (Previous work on this had to be stopped due to struggles with the Softaculous. We will complete this feature at a later moment, potentially based on Installatron.)
```
Comment (AvOeffelen) [inline _site/roadmap.html:238]: easily? XD

**Review by AvOeffelen — CHANGES_REQUESTED** [review]

File: _site/roadmap.html (hunk 4 above)
Comment (cornedenhollander) [reply _site/roadmap.html:238]: Yes, that is an actual word. Suck it.

Comment (AvOeffelen) [reply _site/roadmap.html:238]: Yes i know, but it wasnt that easy to implement Kappa

**Review by AvOeffelen — APPROVED** [review]

**Pull request merged**